src/PointwiseFunctions/GeneralRelativity/WeylElectric.cpp

```diff
@@ -49,6 +49,40 @@ void weyl_electric(
     }
   }
 }
+
+template <size_t SpatialDim, typename Frame, typename DataType>
+Scalar<DataType> weyl_electric_scalar(
+    const tnsr::ii<DataType, SpatialDim, Frame>& weyl_electric,
+    const tnsr::II<DataType, SpatialDim, Frame>&
+        inverse_spatial_metric) noexcept {
+  Scalar<DataType> weyl_electric_scalar_part{};
+  weyl_electric_scalar<SpatialDim>(make_not_null(&weyl_electric_scalar_part),
+                                   weyl_electric, inverse_spatial_metric);
+  return weyl_electric_scalar_part;
+}
+
+template <size_t SpatialDim, typename Frame, typename DataType>
+void weyl_electric_scalar(
+    const gsl::not_null<Scalar<DataType>*> weyl_electric_scalar_part,
+    const tnsr::ii<DataType, SpatialDim, Frame>& weyl_electric,
+    const tnsr::II<DataType, SpatialDim, Frame>&
+        inverse_spatial_metric) noexcept {
+  *weyl_electric_scalar_part =
+      make_with_value<Scalar<DataType>>(get<0, 0>(inverse_spatial_metric), 0.0);
+  for (size_t i = 0; i < SpatialDim; ++i) {
```

Comment:
this can be made more efficient by either raising indices and then contracting or taking into account symmetries